restcountries/base.py

```diff
@@ -3,7 +3,7 @@
 
 
 class RestCountryApi:
-    BASE_URI = "https://restcountries.eu/rest/v1"
+    BASE_URI = "https://restcountries.com/v1"
```

Comment:
just to let you know that V1 does not exists

Reply:
@amatosg thanks for letting us now. I guess we could remove the support for v1 then.